velox/expression/CastExpr.h

```diff
@@ -189,6 +189,11 @@ class CastExpr : public SpecialForm {
       exec::EvalCtx& context,
       const TypePtr& fromType);
 
+  VectorPtr castIntToTimestamp(
+      const SelectivityVector& rows,
+      const BaseVector& input,
+      exec::EvalCtx& context);
+
```

Comment:
We need to update relating documentations.
For Presto, https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/presto/conversion.rst.
For Spark, https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/conversion.rst.
If Presto and Spark behave the same, we can just update the documentation for Presto, and do not need to make a copy for Spark.